packages/init/src/steps/install-lib.ts

```diff
@@ -75,7 +79,7 @@ export async function installLib(
     if (pkgJson instanceof Error) throw pkgJson;
 
     const dependencies = Dependencies.fromPkgFormat(pkgJson.dependencies);
```

Comment:
I think we had a discussion with @hypervillain on this no ? If it's clear for you now can you remove the comment ?